app/assets/stylesheets/components/_autocomplete.scss

```diff
@@ -0,0 +1,7 @@
+@import "accessible-autocomplete/dist/accessible-autocomplete.min.css";
```

Comment:
Do you know if this definitely imports correctly? in Content Publisher we don't use the extension of ".css" and I'm not sure whether that triggers a CSS import

Reply:
I wrote it first like in the example you gave and it worked in the browser but raised a linting error, so I tried it with the `.css` extension and both worked.

Reply:
Looking on the heroku deploy I can see it comes out as:

```
@import url(/assets/smartanswers/accessible-autocomplete/dist/accessible-autocomplete.min-53a3e4a863bb49fa038824ac9bb9f71099b3ca20b68ab846f6942c700da65770.css);.govuk-button,.govspeak .gem-c-button,.gem-c-govspeak .gem-c-button{font-family:"GDS Transport",arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;font-weight:400;font-size:16px;font-size:1rem;line-height:1.1875;box-sizing:border-box;display:inline-block;position:relative;width:100%;margin-top:0;margin-right:0;margin-left:0;margin-bottom:22px;padding:8px 10px 7px;border:2px solid transparent;border-radius:0;color:#fff;background-color:#00703c;box-shadow:0 2px 0 #002d18;text-align:center;vertical-align:top;cursor:pointer;-webkit-appearance:none}/*! 
```

Reply:
That doesn't look great... I've changed it back to just `.min` and put in a linter disable rule.

Reply:
Super - yeah my understanding is that adding ".css" converts it to a CSS import - which is rarely what you want.